src/core/Akka.Streams/Implementation/Fusing/Ops.cs

```diff
@@ -2559,12 +2559,13 @@ public void SetElement(Result<T> result)
                 }
             }
 
-            private static readonly Result<TOut> NotYetThere = Result.Failure<TOut>(new Exception());
+            private static readonly Exception NotYetThereException = new Exception();
+            private static readonly Result<TOut> NotYetThere = Result.Failure<TOut>(NotYetThereException);
 
             private readonly SelectAsync<TIn, TOut> _stage;
             private readonly Decider _decider;
 
-            private IBuffer<Holder<TOut>>? _buffer;
+            private IBuffer<Holder<TOut>> _buffer = null!; // gets initialized in PreStart
```

Comment:
I tried but the downstream `_buffer` calls still complained about nullability anyway 🤷